Add Reissner-Nordstrom metric

AlnesrAltair opened the pull request:
Addresses #525 | Adds support for Reissner-Nordstrom metric

@JeS24

AlnesrAltair committed:
adding reissner-nordstrom metric

Adding module containing reissner-nordstrom metric class and accompanying test file

AlnesrAltair committed:
Updating metric __init__.py

Updating __init__.py in the metric module to reference the Reissner-Nordstrom metric class

pep8speaks commented:
Hello @AlnesrAltair! Thanks for updating this PR. We checked the lines you've touched for [PEP 8](https://www.python.org/dev/peps/pep-0008) issues, and found:



* In the file [`src/einsteinpy/metric/reissnernordstrom.py`](https://github.com/einsteinpy/einsteinpy/blob/0eadba0e89dfc76724adf370a6636e12b2f792bb/src/einsteinpy/metric/reissnernordstrom.py):

> [Line 258:13](https://github.com/einsteinpy/einsteinpy/blob/0eadba0e89dfc76724adf370a6636e12b2f792bb/src/einsteinpy/metric/reissnernordstrom.py#L258): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator
> [Line 259:13](https://github.com/einsteinpy/einsteinpy/blob/0eadba0e89dfc76724adf370a6636e12b2f792bb/src/einsteinpy/metric/reissnernordstrom.py#L259): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator
> [Line 260:13](https://github.com/einsteinpy/einsteinpy/blob/0eadba0e89dfc76724adf370a6636e12b2f792bb/src/einsteinpy/metric/reissnernordstrom.py#L260): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator

* In the file [`tests/test_metric/test_reissnernordstrom.py`](https://github.com/einsteinpy/einsteinpy/blob/0eadba0e89dfc76724adf370a6636e12b2f792bb/tests/test_metric/test_reissnernordstrom.py):

> [Line 40:57](https://github.com/einsteinpy/einsteinpy/blob/0eadba0e89dfc76724adf370a6636e12b2f792bb/tests/test_metric/test_reissnernordstrom.py#L40): [E231](https://duckduckgo.com/?q=pep8%20E231) missing whitespace after ','
> [Line 96:56](https://github.com/einsteinpy/einsteinpy/blob/0eadba0e89dfc76724adf370a6636e12b2f792bb/tests/test_metric/test_reissnernordstrom.py#L96): [E231](https://duckduckgo.com/?q=pep8%20E231) missing whitespace after ','
> [Line 96:78](https://github.com/einsteinpy/einsteinpy/blob/0eadba0e89dfc76724adf370a6636e12b2f792bb/tests/test_metric/test_reissnernordstrom.py#L96): [E231](https://duckduckgo.com/?q=pep8%20E231) missing whitespace after ','



##### Comment last updated at 2024-09-24 09:30:31 UTC

AlnesrAltair committed:
Updating for pep8 compliance

Making style changes without any functionality change to correct pep8 violations

AlnesrAltair committed:
Adding tox reformating

Reformatting changed files using tox. Note that some of the changes conflict with the PEP8 suggestions generated by the repository's checks.

AlnesrAltair commented:
The CircleCI tests are failing on files which have not been altered in this PR.

JeS24 commented:
@AlnesrAltair Ignore the CircleCI tests. The config is out of date. Also, thanks for the PR. I will review it over the next weekend. I'm quite busy till then.

AlnesrAltair commented:
@JeS24 Pls let me know if there anything I can do on this
